demos/dashy/src/dashComms.ts

```diff
@@ -1,19 +1,21 @@
-import { Connection, hookSend, IOptions, ResponseType, SendFunc, serializeRequest } from "@hpcc-js/comms";
+import { Connection, hookSend, IOptions, ResponseType, serializeRequest } from "@hpcc-js/comms";
 
-const origSend = hookSend((opts: IOptions, action: string, request: any, responseType: ResponseType): Promise<any> => {
-    if (opts.baseUrl === "https://webmiscdev.risk.regn.net") {
-        return origSend(opts, action, request, responseType);
-    }
-    let newUrl = "";
-    if (opts.baseUrl.split("").reverse()[0] === "/" || action[0] === "/") {
-        newUrl = btoa(`${opts.baseUrl}${action}?${serializeRequest(request)}`);
-    } else {
-        newUrl = btoa(`${opts.baseUrl}/${action}?${serializeRequest(request)}`);
-    }
-    const connection = new Connection({ baseUrl: "https://webmiscdev.risk.regn.net", type: "get" });
-    return connection.send("brundajx/DASH2/demos/dashy/bis_proxy.php", { encoded: newUrl }, responseType).then(response => {
-        return response;
-    }).catch(e => {
-        throw e;
+export function hookComms() {
+    const origSend = hookSend((opts: IOptions, action: string, request: any, responseType: ResponseType): Promise<any> => {
+        if (opts.baseUrl === "https://webmiscdev.risk.regn.net") {
```

Comment:
These references to `webmiscdev.risk.regn.net` should either be removed or replaced with `webmiscdev01.risk.regn.net`